geonetwork/geonetwork.properties

```diff
@@ -47,7 +47,7 @@ ldap.sync.startDelay=60000
 ldap.privilege.import=true
 ldap.privilege.search.group.queryprop=member
 ldap.privilege.search.group.query=(&(objectClass=groupOfMembers)(member=uid={0},${ldap.base.search.base},${ldap.base.dn})(cn=EL_*))
-ldap.privilege.search.group.object=ou=groups
+ldap.privilege.search.group.object=ou=roles
 ldap.base.search.base=ou=users
 ldap.privilege.search.privilege.pattern=SV_(.*)
 ldap.privilege.search.privilege.query=(&(objectClass=groupOfMembers)(member=uid={0},${ldap.base.search.base},${ldap.base.dn})(cn=SV_*))
```

Comment:
Some comments with "group" are still present in `geonetwork/config/config-security-georchestra.xml`.